code/modules/jobs/job_types/roguetown/garrison/manorguard.dm

```diff
@@ -7,7 +7,8 @@
 	total_positions = 8
 	spawn_positions = 8
 	allowed_sexes = list(MALE, FEMALE)
-	allowed_races = RACES_SHUNNED_UP
+	allowed_races = RACES_ALL_KINDS	//squires can be drow, but now we can do this while disallowing goblins
+	disallowed_races = list(RACES_CONSTRUCT, RACES_GREENSKIN)
```

Comment:
I think adding vermin here is a good call, my PR added kobolds and vermin to despised which some might say was too far, but I don't think they're a good fit for MAA

Reply:
> i like this too I just thought people might crucify me for it tentatively done, same with sergeant!

As someone who originally only main'd Verminvolk on Ratwood, this is actually why I really enjoyed Bogmen for being the designated job you could play as a half-orcs, verminvolk, and kobolds. This gives Wardens on our server more of that 'downtrodden guard' vibe that can turn into 'guard of the downtrodden' if things go south.